astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
+        '''
+        Calculate the parallactic angle.
+
+        Parameters
+        ----------
+        time : `~astropy.time.Time`
+            The time at which the observation is taking place.
+
+        target : {`~astroplan.FixedTarget`, `~astropy.coordinates.SkyCoord`}
+            Celestial object of interest.
+
+        Returns
+        -------
+        `~astropy.coordinates.Angle`
+            Parallactic angle
+        '''
+
+        if not (hasattr(target, 'ra') or hasattr(target, 'dec') or
+                hasattr(target, 'coord')):
+            raise TypeError('The target must be a coordinate (i.e. a '
+                            'FixedTarget or SkyCoord).')
+
+        if hasattr(target, 'coord'):
+            coordinate = target.coord
+        else:
+            coordinate = target
+
+        # Eqn (14.1) of Meeus' Astronomical Algorithms
```

Comment:
Yeah, I agree it would be better if there was something more easily accessed, but unfortunately I don't know of any other equally complete online text comparable to Meeus.  Meeus is one of the more standard texts on spherical trig, though, so I think it's ok.  You could probably also point to http://www.petermeadows.com/html/parallactic.html but I don't think it's as clear as Meeus (and not quite in the same mathematical form)